src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -308,7 +308,7 @@
                   (item "synopsis")
                   (item "createDate")
                   (when (= :config (:type item))
-                    (mapv (get item "method" {}) ["namespace" "name" "snapshotId"]))])}]))
+                    (mapv (get item "method" {}) ["name" "namespace" "snapshotId"]))])}]))
```

Comment:
This isn't for the name and namespace columns, this is for the "Referenced Method" column.